adam-cli/src/test/scala/FlagStatTest.scala

```diff
@@ -0,0 +1,136 @@
+package org.bdgenomics.adam.cli
+import org.apache.spark.rdd.RDD
+
+import org.bdgenomics.adam.projections.{AlignmentRecordField, Projection}
+import org.bdgenomics.adam.rdd.ADAMContext._
+import org.bdgenomics.adam.rdd.read.{DuplicateMetrics, FlagStatMetrics}
+import org.bdgenomics.adam.util.SparkFunSuite
+import org.bdgenomics.formats.avro.AlignmentRecord
+import org.bdgenomics.adam.rdd.read.FlagStat._
+
+class flagstat_test extends SparkFunSuite with java.io.Serializable   {
```

Comment:
Can you change the class name to `FlagStatTest`? We use [PascalCase](http://c2.com/cgi/wiki?PascalCase) for class names and [camelCase](http://c2.com/cgi/wiki?CamelCase) for variable names.

Also, I believe that you shouldn't need to extend `java.io.Serializable` here. With Spark, you should only need your class to be serializable if:

1. It is an object that will be serialized for distribution (e.g., we create RDDs of `AlignmentRecords`, thus `AlignmentRecord` must be serializable)
2. It contains methods that will be executed as part of a distributed action (e.g., inside of a `map`/`flatMap`/etc operation on an `RDD`)